client/my-sites/customer-home/cards/launchpad/pre-launch.tsx

```diff
@@ -16,26 +13,18 @@ const LaunchpadPreLaunch = ( props: LaunchpadPreLaunchProps ): JSX.Element => {
 	const siteId = useSelector( getSelectedSiteId ) || 0;
 	const site = useSelector( ( state: AppState ) => getSite( state, siteId ) );
 	const checklistSlug = site?.options?.site_intent ?? '';
-	const { data: allDomains = [] } = useGetDomainsQuery( site?.ID ?? null, {
-		retry: false,
-	} );
+	const dispatch = useDispatch();
 
-	const layout = useHomeLayoutQuery( siteId || null );
-	const { isOpen, setModalIsOpen, handleSiteLaunched } = useCelebrateLaunchModal( siteId, layout );
+	const handleSiteLaunched = () => {
+		dispatch( launchSiteOrRedirectToLaunchSignupFlow( siteId, 'home' ) );
+	};
 
 	return (
 		<>
 			<CustomerHomeLaunchpad
 				checklistSlug={ props.checklistSlug ?? checklistSlug }
-				onSiteLaunched={ () => handleSiteLaunched( !! site?.is_wpcom_atomic ) }
+				onSiteLaunched={ handleSiteLaunched }
 			/>
-			{ isOpen && (
-				<CelebrateLaunchModal
-					setModalIsOpen={ setModalIsOpen }
-					site={ site }
-					allDomains={ allDomains }
-				/>
-			) }
```

Comment:
Same thing here: Since we now have CelebrateLaunchModal in the top Home component, we no longer need to render and control it individually for each Launchpad task.